db/db_impl/db_impl.h

```diff
@@ -1956,6 +1958,7 @@ class DBImpl : public DB {
   void ReleaseFileNumberFromPendingOutputs(
       std::unique_ptr<std::list<uint64_t>::iterator>& v);
 
+  // Sets bg error if there is an error writing to WAL.
```

Comment:
"// Sets bg error if there is an error writing to WAL." is this comment for SyncClosedWals or WALIOStatusCheck?

Reply:
It's meant for SyncClosedWals(). Just to emphasis that this function will set bg error.